src/query/insert.rs

```diff
@@ -173,6 +222,7 @@ where
     /// };
     /// assert_eq!(
     ///     cake::Entity::insert(orange)
+    ///         .on_empty_do_nothing()
```

Comment:
umm, why do we need to change the examples here? I think this PR should have no changes to exsiting features